### PR TITLE
Addresses #2020 NTR alveolar macrophages

### LINKED ISSUE #2020: [NTR] Pulmonary macrophages
**Preferred term label**
alveolar macrophage CCL3-positive

**Synonyms** (add reference(s), please)

**Definition** 
An alveolar macrophage that expresses CCL3.
doi:10.1101/2022.03.10.483747

**Parent cell type term**
'alveolar macrophage' CL:0000583

**What is the anatomical structure that the cell is a part of?** 
It will be inherited

**Your ORCID**
https://orcid.org/0000-0001-6677-8489
https://orcid.org/0000-0002-9185-3994

**Additional notes or concerns**

-------

**Preferred term label**
alveolar macrophage metallothionein-positive

**Synonyms** (add reference(s), please)
alveolar macrophage MT-positive
doi:10.1101/2022.03.10.483747

**Definition** 
An alveolar macrophage that expresses metallothionein.
doi:10.1101/2022.03.10.483747

**Parent cell type term**
alveolar macrophage

**What is the anatomical structure that the cell is a part of?** 
It will be inherited.

**Your ORCID**
https://orcid.org/0000-0001-6677-8489
https://orcid.org/0000-0002-9185-3994

**Additional notes or concerns**

-------
**Preferred term label**
lung interstitial macrophage

**Synonyms** (add reference(s), please)
lung IM (abbreviation) PMID:[36211428](https://pubmed.ncbi.nlm.nih.gov/36211428)

**Definition** 
A macrophage that is part of the lung connective tissue (pulmonary interstitium). This cell performs tissue remodeling and contributes to barrier immunity through antigen presentation. 
PMID:[33382972](https://pubmed.ncbi.nlm.nih.gov/33382972)
PMID:[36211428](https://pubmed.ncbi.nlm.nih.gov/36211428)
PMID:[35017701](https://pubmed.ncbi.nlm.nih.gov/35017701)

**Parent cell type term**
macrophage ('lung macrophage' should be inferred)

**What is the anatomical structure that the cell is a part of?** 
lung connective tissue (UBERON:0000114)

**Your ORCID**
https://orcid.org/0000-0001-6677-8489
https://orcid.org/0000-0002-9185-3994

**Additional notes or concerns**

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #2020 NTR alveolar macrophages

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -31484,6 +31487,36 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27776108") oboInOwl:hasE
 AnnotationAssertion(rdfs:label obo:CL_4033040 "lung resident memory CD8-positive, CD103-positive, alpha-beta T cell")
 EquivalentClasses(obo:CL_4033040 ObjectIntersectionOf(obo:CL_4033039 ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000001010)))
 
+# Class: obo:CL_4033041 (alveolar macrophage CCL3-positive)
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:31490]: Labels in CL tend to have markers at the start of the label, so consider relabelling to 'CCL3-positive alveolar macrophage'.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -31484,6 +31487,36 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27776108") oboInOwl:hasE
...
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37291214") obo:IAO_0000115 obo:CL_4033041 "An alveolar macrophage that expresses CCL3.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033041 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033041 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033041 "2023-06-15T12:53:08Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4033041 "alveolar macrophage CCL3-positive")
+SubClassOf(obo:CL_4033041 obo:CL_0000583)
+
+# Class: obo:CL_4033042 (alveolar macrophage metallothionein-positive)
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:31499]: Labels in CL tend to have markers at the start of the label, so consider relabelling to 'metallothionein-positive alveolar macrophage'.

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -31484,6 +31487,36 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27776108") oboInOwl:hasE
 AnnotationAssertion(rdfs:label obo:CL_4033040 "lung resident memory CD8-positive, CD103-positive, alpha-beta T cell")
 EquivalentClasses(obo:CL_4033040 ObjectIntersectionOf(obo:CL_4033039 ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000001010)))
 
+# Class: obo:CL_4033041 (alveolar macrophage CCL3-positive)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37291214") obo:IAO_0000115 obo:CL_4033041 "An alveolar macrophage that expresses CCL3.")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:31492]: Consider adding logical statement to represent "expresses CCL3".
If there are any other distinguishing features (e.g., marker genes) or functions ((e.g., see doi:[10.1128/IAI.69.10.6364-6369.2001](https://doi.org/10.1128/iai.69.10.6364-6369.2001)) consider adding.

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -31484,6 +31487,36 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27776108") oboInOwl:hasE
...
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033041 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033041 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033041 "2023-06-15T12:53:08Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4033041 "alveolar macrophage CCL3-positive")
+SubClassOf(obo:CL_4033041 obo:CL_0000583)
+
+# Class: obo:CL_4033042 (alveolar macrophage metallothionein-positive)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37291214") obo:IAO_0000115 obo:CL_4033042 "An alveolar macrophage that expresses metallothionein.")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:31501]: If there are any other distinguishing features (e.g., marker genes) or functions consider adding to definition or comment.

**Review by bvarner-ebi — COMMENTED** [review]
Cursory content review performed. Defer content expertise to requester.

No breaking changes noted. 

Kindly see inline comments.

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:31492]: Thank you for your comment. The ontology that is normally used is PR, but in this case the [CCL3 (human)](https://www.ebi.ac.uk/ols4/ontologies/pr/classes/http%253A%252F%252Fwww.genenames.org%252Fcgi-bin%252Fgene_symbol_report%253Fhgnc_id%253D10627) term is imported, and I am not sure if I should use it.
I have added the markers that the reference provided in Supplementary Table 6 (see screenshot):
image

Interestingly, these markers differ from other sources (eg. Azimuth):
image

> If there are any other distinguishing features (e.g., marker genes) or functions ((e.g., see doi:[10.1128/IAI.69.10.6364-6369.2001](https://doi.org/10.1128/iai.69.10.6364-6369.2001)) consider adding to definition or comment.

I am reluctant to use conclusions of this study in this term. In the study, authors use KO mice, meaning that they are comparing alveolar macrophages WT mice, with alveolar macrophages CCL3-/-. The conclusion is that KO mice have impaired phagocytic activity respect WT. In my opinion, it doesn't mean that this cell has more phagocytic activity than other WT alveolar macrophages, as that could only be demonstrated using cell sorting from a WT mouse and test the phagocytic activity. As I might have missed something from the article, please let me know if you agree with my interpretation.

File: src/ontology/cl-edit.owl (hunk 4 above)
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:31501]: Thank you for your comment. 
I have added the markers that the reference provided in Supplementary Table 6 (see screenshot):
image
Interestingly, these markers differ from other sources (eg. Azimuth):
image

**Review by bvarner-ebi — APPROVED** [review]

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Pull request merged**